scripts/profile-compiler-build.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/sh
 
-# To be run from the root of the Flambda backend repo
-
 set -e -u -o pipefail
 
-dump_dir="`pwd`/_profile"
-summary_path="`pwd`/summary.csv"
+# Works regardless of where script run from
+scripts_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
```suggestion
scripts_dir=$( cd "$( dirname "$0" )" &> /dev/null && pwd )
```
Is that version enough to achieve what we want?
(I am just worried the current version is less
compatible with various shells.)

Reply:
Yes, this is sufficient.